Article/Article.js

```diff
@@ -3,24 +3,26 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = this.domElement.querySelector('.expandButton')
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    this.expandButton.innerText = 'Expand'
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener('click', () => {
```

Comment:
if you want to pass an event to expandArticle, you need to do:
```
(event) => {
  this.expandArticle(event);
}
```

Reply:
but you only need to pass an event if you make use of it, like `event.stopPropagation()`, which might not be a bad idea